src/Main.java

```diff
@@ -25,49 +25,68 @@ public static void main(String[] args) {
         byte m = 67;
 
         // Задача 3
-        byte class1 = 23;
-        byte class2 = 27;
-        byte class3 = 30;
-        short paper = 480;
+        int class1 = 23;
+        int class2 = 27;
+        int class3 = 30;
+        int paper = 480;
         System.out.println("На каждого ученика рассчитано "+(paper /(class1 + class2 + class3))+ " листов бумаги");
 
         // Задача 4
-        byte bottle = 16;
-        byte time = 2;
-        System.out.println("За 20 минут машина произвела "+ bottle /time *20 +" штук бутылок");
-        System.out.println("За сутки машина произвела "+ bottle /time *24 *60 +" штук бутылок");
-        System.out.println("За 3 дня машина произвела "+ bottle /time *3 *24 *60 +" штук бутылок");
-        System.out.println("За 1 месяц машина произвела "+ bottle/time *30 *24 *60 +" штук бутылок");
+        int bottle = 16;
+        int time = 2;
+        int min = bottle /time;
+        int min20 = min *20;
+        System.out.println("За 20 минут машина произвела "+ min20 +" штук бутылок");
+        int day = min *60 *24;
+        System.out.println("За сутки машина произвела "+ day +" штук бутылок");
+        day = day *3;
+        System.out.println("За 3 дня машина произвела "+ day +" штук бутылок");
+        day = day *30;
+        System.out.println("За 1 месяц машина произвела "+ day +" штук бутылок");
 
         // Задача 5
-        byte remont = 120;
-        byte white = 2;
-        byte brown = 4;
+        int remont = 120;
+        int white = 2;
+        int brown = 4;
         int classes = remont /(white +brown);
-        System.out.println("В школе, где "+ classes +" классов, нужно "+ classes *2 + " банок белой краски и "+ classes *4 +" банок коричневой краски");
+        white = classes *2;
+        brown = classes *4;
+        System.out.println("В школе, где "+ classes +" классов, нужно "+ white + " банок белой краски и "+ brown +" банок коричневой краски");
 
         // Задача 6
         int bananas = 5 *80;
         int milks = 200 /100 *105;
         int iceCreams = 2 *100;
         int aggs = 4 *70;
-        float breakfast = (float) (bananas +milks +iceCreams +aggs);
-        System.out.println("Вес завтрака спортсмена составляет "+ breakfast +" г или "+ (breakfast /1000) +" кг");
+        int breakfast = (bananas +milks +iceCreams +aggs);
+        double breakfast1 = breakfast /1000;
```

Comment:
деление с потерей остатка. Т.к. у тебя оба операнда деления целые числа, то деление будет целым. Т.е. 5 / 2 = 2, а не 2.5. Чтобы получилось вещественным, нужно чтобы хотя бы 1 из операндов был вещественным, например, так
```suggestion
        double breakfast1 = breakfast /1000D;
```